src/test/resources/features/CheckYourAnswers.feature

```diff
@@ -3,48 +3,49 @@ Feature: Check Your Answers
 
```

Comment:
@warrenskiwork1979 We'll need to add the @Registration tag to the top of this file to include it in the tests

Reply:
Currently duplicating what Registration journey does and needs more work.

Reply:
Thanks Warren, just noticed the tag was missing the other day and assumed we were trying to put it back in now 😄 I'll see if I can get any time to have a proper look at them today/Monday!